geoh5py/shared/merging/drape_model.py

```diff
@@ -139,32 +139,30 @@ def merge_data(
     ):
         super().merge_data(out_entity, input_entities)
 
-        # prepare an empty nan array
-        values: np.ndarray = np.empty(out_entity.n_cells)
-        values[:] = np.nan
+        ind_map = []
+        data_count = 0
+        ghost_count = 0
+        n_values = np.sum([input_entity.n_cells for input_entity in input_entities])
+        for input_entity in input_entities:
+            if input_entity.n_cells is None:
```

Comment:
Is this possible? We check for prisms and layers to exist for input entities, 
except for pylint, we don't need that, doesn't we? cast does the job.

Reply:
It is possible that one of the input object doesn't have cell defined.

Reply:
If so, it raises an error at the beginning no?
It checks for every object to have prisms and layers, which are associated to cells
Except if prisms and layers are empty array? I don't even know if it's possible.

Reply:
Correct, technically it is checked in during the super(). And yes, it is raised by pylint since n_cells can be None.